workers/unity/Packages/com.improbable.gdk.core/Utility/EntityBuilder.cs

```diff
@@ -5,6 +5,9 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Enables the creation of entity templates with a fluent interface.
```

Comment:
opportunity to link to fluent interface definition here

Reply:
also cc @paulbalaji shall we update `EntityBuilder` docs to mention "fluent interface" instead of "builder pattern"? FI seems more appropriate

Reply:
How does one do http links unobtrusively? I've only added them where absolutely necessary as they don't render nicely

Reply:
if there's no nice easy way it's k

Reply:
A fluent interface is a just a way to implement a builder.